pjlib/src/pj/string.c

```diff
@@ -437,6 +437,72 @@ PJ_DEF(pj_status_t) pj_strtoul3(const pj_str_t *str, unsigned long *value,
     return PJ_SUCCESS;
 }
 
+PJ_DEF(pj_status_t) pj_strtoul4(const pj_str_t *str, pj_uint_t *value,
+                                unsigned base)
```

Comment:
can strtoul3() call this, to minimise duplication?

Reply:
Perhaps yes, however as they are actually different (long can be 32 bit or more), it should be safe if we assume that:
- if a system does not have 64 bit, long will always be 32 bit,
- `long` is always in the range of 32-64 bit (the spec seems to only specify minimum 32 bit), if some system's `long` is more than 64 bit, it is vice versa (strtoul4() should call strtoul3) :)